src/Mod/PartDesign/App/FeatureExtrude.h

```diff
@@ -24,12 +24,17 @@
 #ifndef PARTDESIGN_FEATURE_EXTRUDE_H
 #define PARTDESIGN_FEATURE_EXTRUDE_H
 
+#include <App/PropertyGeo.h>
 #include <App/PropertyStandard.h>
 #include <App/PropertyUnits.h>
+#include <Base/Tools.h>
 #include "FeatureSketchBased.h"
 
 namespace PartDesign
 {
+    static const App::PropertyQuantityConstraint::Constraints signedLengthConstraint = { -DBL_MAX, DBL_MAX, 1.0 };
+    static const double maxAngle = 90 - Base::toDegrees<double>(Precision::Angular());
```

Comment:
By moving this from the .cpp to the .h file (compared to the [initial commit](https://github.com/FreeCAD/FreeCAD/commit/821b330722a361e71632fa49a17e2cc76e1e6e69)), you now need `#include <Precision.hxx>` in the header, too. Otherwise it fails to compile here using apple-clang-11.0

Reply:
OK, so not it is clear to me that the CI does not use precompiled headers. Since I do I don't see the compilation failures.
Now it should work, please report back.